x-pack/plugins/apm/public/components/shared/charts/transaction_charts/helper.tsx

```diff
@@ -24,12 +23,19 @@ export function getResponseTimeTooltipFormatter(formatter: TimeFormatter) {
   };
 }
 
-export function getMaxY(timeSeries: TimeSeries[]) {
-  const coordinates = flatten(
-    timeSeries.map((serie: TimeSeries) => serie.data as Coordinate[])
-  );
+export function getMaxY(
+  timeSeries?: Array<TimeSeries<Coordinate>> | TimeSeries<Coordinate>
+) {
+  if (timeSeries) {
+    const coordinates = Array.isArray(timeSeries)
+      ? timeSeries.flatMap((serie) => serie.data)
+      : timeSeries.data;
 
-  const numbers: number[] = coordinates.map((c: Coordinate) => (c.y ? c.y : 0));
+    const numbers: number[] = coordinates.map((c: Coordinate) =>
```

Comment:
These tyes should not be needed

```suggestion
    const numbers = coordinates.map((c) =>
```